test/sasdataloader/test/utest_abs_reader.py

```diff
@@ -85,8 +86,12 @@ def test_hfir_checkdata(self):
 class igor_reader(unittest.TestCase):
     
     def setUp(self):
-        self.data = Loader().load("MAR07232_rest.ASC")
-        
+        # the IgorReader should be able to read this filetype
+        # if it can't, stop here.
+        reader = IgorReader()
+        output = reader.read("MAR07232_rest.ASC")
+        self.assertEqual(output.filename, "MAR07232_rest.ASC")
```

Comment:
This same assertion is already done in test_igor_checkdata, but with a data set loaded using the generic dataloader. A better test might be comparing a data file loaded with Loader.load() to the same data file loaded with IgorReader.read().